P9Operators/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿/* P9Operators main exercise
```

Comment:
Unfortunately, all of your code is commented out. Make sure to put the code back into action again. Technically, right now, your operators exercise doesn't work as expected. But I'll ignore it this time :)